boxes/README.md

```diff
@@ -0,0 +1,36 @@
+# Vagrant box generation
+
+This section defines the procedure to build Vagrant boxes required for WSO2 Identity Server 5.4.0 single node
+deployment with Analytics support.
+
+Please note that in order to build these Vagrant boxes, you need to install
+[Oracle VM VirtualBox](http://www.oracle.com/technetwork/server-storage/virtualbox/downloads/index.html),
+as WSO2 Vagrant resources use Oracle VM VirtualBox, as the default provider.
+
```

Comment:
Since Ruby is a dependency, we should mention it as well

Reply:
+1 @ThilinaManamgoda, in order to run `box_definitions.rb` we need Ruby.